noxfile.py

```diff
@@ -53,14 +53,17 @@ def docs(session: nox.Session) -> None:
 
     session.install("-r", "docs/requirements.txt")
     session.chdir("docs")
-    session.run("sphinx-build", "-M", "html", ".", "_build")
-
-    if session.posargs:
-        if "serve" in session.posargs:
-            print("Launching docs at http://localhost:8000/ - use Ctrl-C to quit")
-            session.run("python", "-m", "http.server", "8000", "-d", "_build/html")
-        else:
-            print("Unsupported argument to docs")
+    if "pdf" in session.posargs:
+        session.run("sphinx-build", "-M", "latexpdf", ".", "_build")
+        return
+    else:
```

Comment:
else return. I should add flake8 module for this as well.

Reply:
🤦